tests/integration/synapseutils/test_synapseutils_copy.py

```diff
@@ -309,85 +309,3 @@ def test_entitySubPageId_and_destinationSubPageId(self):
         assert_equals(temp.title, self.sub_subwiki.title)
         assert_equals(temp.markdown, self.sub_subwiki.markdown)
 
-
```

Comment:
add 1 integration test that make copy of 2 file handles

Reply:
recommandation:
1. in the test setup, create an external file handle (URL to files on the internet) (you can use this URL: https://www.synapse.org/images/logo.svg) Either use existing synapse client function or https://docs.synapse.org/rest/POST/externalFileHandle.html to create external file handle
2. make copies
You don't need to clean up these file handles -- these will be clean up every week when we relaunch the dev stack.